src/glados/engine/tui_integration.py

```
@@ -0,0 +1,193 @@
+        if reasoning_step:
+            self.notify_tui("chain_of_thought", {
+                "step": reasoning_step,
+                "reasoning": chunk,
+                "timestamp": time.strftime("%H:%M:%S")
+            })
+
+    def _notify_tool_calling(
+        self, 
+        tool_name: str, 
+        status: str, 
+        parameters: Dict[str, Any] = None,
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)